### ISSUE TITLE
🍺 [CLI] Run assessment: Export result to new file /Jenkins/

### ISSUE THREAD

**Issue by davidmoremad** [Telefonica/Dome9#13]
Add another option on `run_assessment` in order to export results to a new file so Devops can handle this report easier on Jenkins job.

**Comment by davidmoremad** [comment]
Closed on Telefonica/dome9-cli@a84c4db0ddf07d91067f0c82d80090b489e7c266

New argument `--export`
```
$ dome9cli.py get_assessment 0123456789 --export

---------------- Export ----------------
[i] Report saved: ./assessment.json
[ ] File size: 5357 KB

---------------- Stats ----------------
...
```

**Comment by davidmoremad** [comment]
https://github.com/Telefonica/dome9-cli/commit/a84c4db0ddf07d91067f0c82d80090b489e7c266
